### PR TITLE
Added New System Tests

### PR DESCRIPTION
**Pull request by TZnoj**
Added the System Tests for the features added in Release 3 and the failed system test from Release 2. Modified old system tests to have a working Auth Token.
- US-16: Editing Properties of a Vehicle of a Dealership
- US-18: Download Vehicle QR Code
- US-19: Dashboard Informational Diagrams
- US-20: Adding a Vehicle Zone to a Dealership
- US-21: Modifying a Vehicle Zone
- US-22: Adding a Vehicle to a Vehicle Zone

### CODE REVIEW

**Review by AbdulSir — APPROVED** [review]
Everything looks great. Maybe we can address the the code smell that's flagged by sonarCloud?

File: Jenkinsfile [hunk 1]
```diff
@@ -48,6 +48,7 @@ pipeline {
           steps {
             dir(path: 'Autotracks/frontend') {
               echo 'This will be deployment area'
+              echo 'minor edit to pass check'
```
Comment (Emperor42) [inline Jenkinsfile:51]: will be removed when merging with deployment

**Review by Emperor42 — APPROVED** [review]
Everything looks good, all checks passed great work!

**Review by purplesmurf1998 — APPROVED** [review]

**Review by qandeelarshad — APPROVED** [review]
Everything looks good. Good work!

**Pull request merged**